src/univariate/continuous/normal.jl

```diff
@@ -74,22 +73,18 @@ kurtosis(d::Normal{T}) where {T<:Real} = zero(T)
 
 entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 
-
 #### Evaluation
-
-@_delegate_statsfuns Normal norm μ σ
```

Comment:
Check https://github.com/JuliaStats/Distributions.jl/blob/2ac3a83aef47132d97bb6632e8465617cd493c9f/src/univariates.jl#L529 for a list of the functions which are created by this macro and should be implemented natively.

Reply:
I checked whether including some methods had any noticeable drawback from using the default   fallback and included those only if it applied. For example, I did make one change to `quantile` such that the `invlogcdf` fallback was consistent providing `NaN`.